book/glossary.md

```diff
@@ -10,329 +10,511 @@ clear names*.
+
+: The property of an element (sometimes in the web page,
 sometimes in the browser chrome) being set to receive future keyboard
 events and other user interactions. Typically, the focused element is
 visually highlighted on the screen.
 
-*Font*: A particular stylistic way of drawing a particular human language to
+Font
+
+: A particular stylistic way of drawing a particular human language to
 computer screens. Times New Roman is one common example for Latin-based
 languages.
 
-*HTML attribute*: A parameter on an element indicating some
+HTML attribute
+
+: A parameter on an element indicating some
 information, such as the source of an image or URL of a style sheet.
 
-*Iframe*: A way of embedding one document within another. A
+Iframe
+
+: A way of embedding one document within another. A
 rectangular window in the parent document shows the child document and
 participates in the layout of the parent.
 
-*Image*: A representation of a picture to draw on a computer screen. An
+Image
+
+: A representation of a picture to draw on a computer screen. An
 HTML element of the same name, for the same purpose.
 
-*Node*: A point in the DOM tree, with parent and child pointers.
+Node
+
+: A point in the DOM tree, with parent and child pointers.
 
-*Page*: The conceptual container for a document. A page can have
+Page
+
+: The conceptual container for a document. A page can have
 multiple documents through use of iframes.
 
-*Parsing*: Turning a serialized representation (such as HTML or CSS)
+Parsing
+
+: Turning a serialized representation (such as HTML or CSS)
 into a data structure such as the document tree or a style sheet.
 
-*Style sheet*: A web resource that contains CSS rules.
+Style sheet
 
-*Tag name*: The name of a particular type of HTML element, indicating its
+: A web resource that contains CSS rules.
+
+Tag name
+
+: The name of a particular type of HTML element, indicating its
 semantic function in the document. Usually comes with special style rules
 and functionality specific to it.
 
 Networking
 ==========
 
-*GET*: The mode of HTTP that retrieves a server resource without changing it.
+Cookie
 
-*Cookie*: A piece of persistent, per-site state stored by web browsers
+: A piece of persistent, per-site state stored by web browsers
 to enable use cases like user logged-in status for access-controlled content.
 
-*Domain*: The name of a website, used to locate it on the internet.
+Domain
+
+: The name of a website, used to locate it on the internet.
+
+GET
+
+: The mode of HTTP that retrieves a server resource without changing it.
+
+Path
+
+: The part of a URL after the domain and port.
 
-*Path*: The part of a URL after the domain and port.
+Port
 
-*Port*: A number after the domain and before the path in a URL, indicating
+: A number after the domain and before the path in a URL, indicating
 a numbered place on that domain with which to communicate.
 
-*POST*: The mode of HTTP that submits a change to server state and expects
+POST
```

Comment:
Not in code font? I'm fine either way but I think typically we code-font this.

Reply:
They didn't recommend that...

Reply:
Ok, then leave it as is.